statscache/app.py

```diff
@@ -36,24 +28,36 @@ def main(name):
     )
 
 
+@app.route('/')
+def index():
+    """ Generate a JSON-P response with an index of plugins (as an array) """
+    return jsonp(json.dumps(plugins.keys()), 200)
```

Comment:
Here's a cosmetic request:  instead of putting this on the root url, can we put it on ``/api`` and then put the main endpoint on ``/api/<name>``?  This will allow us to put a static HTML landing page on the root URL so people who visit https://apps.fedoraproject.org/statscache can get a clue about what's going on instead of being presented with JSON right off the bat.